azure-webapp-maven-plugin/src/main/java/com/microsoft/azure/maven/webapp/handlers/DeploymentSlotHandler.java

```diff
@@ -0,0 +1,14 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ */
+
+package com.microsoft.azure.maven.webapp.handlers;
+
+import com.microsoft.azure.maven.auth.AzureAuthFailureException;
+import org.apache.maven.plugin.MojoExecutionException;
+
+public interface DeploymentSlotHandler {
+    void handleDeploymentSlot() throws AzureAuthFailureException, MojoExecutionException;
```

Comment:
what does "handle" indicate?
in the implementation, i only see create if not exists operation. please be explicit if there are no other possibilities.